benchmark/jsonschema/compiler_basic.cc

```diff
@@ -0,0 +1,25 @@
+#include <benchmark/benchmark.h>
+
+#include <sourcemeta/jsontoolkit/json.h>
+#include <sourcemeta/jsontoolkit/jsonschema.h>
+
+static void Compiler_Basic(benchmark::State &state) {
+  for (auto _ : state) {
+    const sourcemeta::jsontoolkit::JSON schema{
```

Comment:
I think this JSON parsing can be outside the measured part, as we are only benchmarking the compiler